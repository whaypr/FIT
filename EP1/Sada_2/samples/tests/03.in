10
60 30
120 28
180 35
240 34
300 40
360 31
420 28
480 2
540 42
600 30
8
lt avg 0
lt avg 100000
lt avg 60
gt avg 150
gt avg 0
gt avg 100000
gt avg 120
gt avg 260
