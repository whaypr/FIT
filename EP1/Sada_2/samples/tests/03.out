0
4
6
5
0
4
5
5
